app/src/main/java/io/neurolab/program_modes/MemoryGraphActivity.java

```diff
@@ -64,6 +66,29 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         setTitle(R.string.mem_graph);
 
         memGraph = findViewById(R.id.mem_graph);
+        BottomNavigationView navigationView = findViewById(R.id.navigation);
+        BottomNavigationItemView selectedMenuItem = findViewById(R.id.bottom_nav_graph);
+        selectedMenuItem.setEnabled(true);
+        selectedMenuItem.setBackgroundColor(getResources().getColor(android.R.color.holo_purple));
+        navigationView.setOnNavigationItemSelectedListener(menuItem -> {
```

Comment:
Heard about Android Fragments? This is not how bottom navigation bar is implemented. Refer to some tutorials on how bottomnavigation is done. https://www.truiton.com/2017/01/android-bottom-navigation-bar-example/

Reply:
Yup @CloudyPadmal but actually implentation of some stuff were restricted in the Fragments life cycle. Do let me if we still want to proceed with Fragments, i will make the changes, also then we need to convert the Memory Graph Activity to Fragment. Do let me know your thoughts, will definitely make the changes.

Reply:
What are those some stuff we cannot implement in fragments? :thinking:

Reply:
Fragments are more favored as you can see everytime we switch a tab, there is a new window opening and screen flashes. With fragments we won't get that effect.

Reply:
Yeah exactly. :sweat_smile:

Reply:
> What are those some stuff we cannot implement in fragments? thinking

Don't remember the specifics, had tried earlier. Nevertheles, will change to fragments and will let you know if any problem arises.